includes/derivatives.inc

```diff
@@ -21,6 +21,18 @@
  * @see hook_islandora_derivative()
  */
 function islandora_fits_create_techmd(AbstractObject $object, $force = FALSE, $hook = array()) {
+  if (!variable_get('islandora_fits_do_derivatives', TRUE)) {
+    return array(
+      'success' => TRUE,
```

Comment:
This is minor, but do you count this as a success?

Reply:
Here's the results of my testing this PR.

System path input is sanitized:
![image](https://user-images.githubusercontent.com/10052068/31355308-b53131a4-ad07-11e7-8354-6205ef7f107e.png)

But an invalid path is still returning the success message:
![image](https://user-images.githubusercontent.com/10052068/31355405-2275cdf6-ad08-11e7-9804-753fac90d617.png)

Reply:
Sorry @mrmiguez I missed this comment.

This function is called as part of the derivative creation process, so I thing the path checking part is fine I'm just wondering if we need to return an response stating that not running FITS because we have chosen not to is a success?

Reply:
Leaving the admin form aside for a moment (because when the red x vs green checkmark shows up is probably worth discussing) let's talk about what the derivatives function should return - you're right, success is probably not it. Maybe we should put the variable check one level up, in hook_islandora_derivative, the way it's done in the Video SP.  (e.g. https://github.com/Islandora/islandora_solution_pack_video/blob/7.x/islandora_video.module#L140) That way, we don't even run the derivative function so don't have to have it return success or failure or anything else.

Reply:
I've removed this in commit https://github.com/Islandora/islandora_fits/pull/64/commits/c80437d1179d5ebe38bd4aa58f171f13cea7cc1a and instead, we test for whether `islandora_fits_do_derivatives` is set in hook_islandora_derivatives. No more returning success when nothing happened.

Reply:
To return to @mrmiguez 's testing (thanks!) `'/var; touch /home/vagrant/tmp'` and `'/var/log/'` now should both return "The specified file path... does not exist."